### PR TITLE
exchanges - promise.all

### PR DESCRIPTION
**Pull request by ttodua**
@carlosmiei I think there opens up a bigger path to use promise.all for all such cases.
i understand that "in most cases" users are reusing the instance, but "as long as we can" why not make promise.all for all such areas? 

this PR just includes most basic cases (where no response is needed atm).

### CODE REVIEW

**Comment by carlosmiei** [comment]
@ttodua I think we can use `promise.all` yes as long as we make sure methods like `fetchAccount` are not using results from `loadMarkets`

File: ts/src/coinbase.ts [hunk 1]
```diff
@@ -2648,8 +2648,7 @@ export default class coinbase extends Exchange {
     }
 
     async findAccountId (code, params = {}) {
-        await this.loadMarkets ();
```
Comment (carlosmiei) [inline ts/src/coinbase.ts:2651]: not sure we should do this here
image

File: ts/src/htx.ts [hunk 2]
```diff
@@ -5075,8 +5075,7 @@ export default class htx extends Exchange {
          * @param {float} [params.cost] the quote quantity that can be used as an alternative for the amount for market buy orders
          * @returns {object} request to be sent to the exchange
          */
-        await this.loadMarkets ();
-        await this.loadAccounts ();
+        await Promise.all ([ this.loadMarkets (), this.loadAccounts () ]);
```
Comment (carlosmiei) [inline ts/src/htx.ts:5078]: same here 
image

File: ts/src/coinbase.ts (hunk 1 above)
Comment (ttodua) [reply ts/src/coinbase.ts:2651]: question here, even if `fetchAccounts` internally use `loadMarkets`, does it matter? what I see, as long as we have safe-thread access to markets loading promise, it shouldn't be a problem. not?

Comment (ttodua) [reply ts/src/coinbase.ts:2651]: ah wait, i missed the point probably, you're right, fAccounts shouldnt be using results from loaded markets. ok, give me a moment to revise them

**Comment by ttodua** [comment]
fixed and reverted all `fetchAccounts` calls in promise all

**Pull request merged**